.github/workflows/ci.yml

```diff
@@ -8,36 +8,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14.x
+      - name: Use Node.js 14.17
         uses: actions/setup-node@v2.1.5
         with:
-          node-version: '14.x'
+          node-version: '14.17'
       - name: Run yarn dedupe
         run: yarn dedupe --check
 
   format:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14.x
+      - name: Use Node.js 14.17
         uses: actions/setup-node@v2.1.5
         with:
-          node-version: '14.x'
+          node-version: '14.17'
       - name: Install dependencies
-        run: yarn install --immutable --immutable-cache
+        run: yarn install
```

Comment:
We should make sure to keep this as `yarn install --immutable --immutable-cache` for CI builds